pep-0678.rst

```diff
@@ -319,15 +319,16 @@ proposed ``__notes__`` semantics, but this would be rarely and inconsistently
 applicable.
 
 
-Store notes in ``ExceptionGroup``\ s, but not other ``Exception``\ s
---------------------------------------------------------------------
-Initial discussions proposed making a more focussed change by thinking about
-how to associate messages with the nested exceptions in ``ExceptionGroup`` s,
-such as a list of notes or mapping of exceptions to notes.  However, this would
-force a remarkably awkward API and retains a lesser form of the
-cross-referencing problem discussed under "use ``print()``" above; if this PEP
-is rejected we prefer the status quo. Finally, of course, ``__notes__`` are
-not only useful with ``ExceptionGroup``\ s!
+Don't attach notes to ``Exception``\ s, just store them in ``ExceptionGroup``\ s
+--------------------------------------------------------------------------------
+The initial motivation for this PEP was to associate a note with each error
+in an ``ExceptionGroup``.  At the cost of a remarkably awkward API and the
+cross-referencing problem discussed under "use ``print()``" above, this
```

Comment:
Especially since it is a number of sections above, I suggest actually linking the appropriate section, e.g. by adding an explicit reference right above the print() section heading, i.e.

```rst
.. _print_idea:

Use print() ...
-------------
```

then adding the appropriate link here, i.e.

```suggestion
cross-referencing problem discussed `above <print_idea_>`_, this
```